packages/cli/src/commands/init/__tests__/template.test.js

```diff
@@ -16,13 +17,14 @@ afterEach(() => {
   jest.restoreAllMocks();
```

Comment:
this doesn't clean the `execa` because it's not a spy. Either move `execa` to spies or call `jest.clearAllMocks` additionally (same for other cases)